### DISCUSSION TITLE
[BUG][CRITICAL] Domain restricted videos play after html page download using 'save as'

### DISCUSSION THREAD

**Discussion by prabushitha** [vimeo/player.js#998] [Bug reports]
### Expected Behavior
Should give an authentication error
```Sorry
Because of its privacy settings, this video cannot be played here.
```

### Actual Behavior
Video can be played by opening the downloaded HTML

### Steps to Reproduce
- Upload a video and set privacy to `Specific Domains`
- Embed the video on a valid domain
- Go to the embedded page in the domain and save the page (right click -> save as)
- Open the download page from the local computer
- Although the video is private, embedded video can be played from the downloaded HTML file.

**Comment by bdougherty** [comment]
@prabushitha if this is still happening, it's probably best to [contact support](https://vimeo.com/help/contact). We at least need the id of the video to investigate.
